atomic

```diff
@@ -299,11 +299,14 @@ def create_parser(help_text):
     installp.add_argument("image", help=_("container image"))
     if Atomic.syscontainers.OSTREE_PRESENT:
         system_xor_user = installp.add_mutually_exclusive_group()
```

Comment:
What happens if both are not available?  Does this work correctly and just ignore the exclusive_group?